netlify.toml

```diff
@@ -4,3 +4,7 @@
   base = "code/"
   publish = "build/"
   command = "npm run build"
+  [[redirects]]
+  from = "/*"
+  to = "/"
+  status = 200 
```

Comment:
😉 💖

Reply:
Overall well done with the project and yes your are awesome in styling , that is for sure. Awesome job.